src/lib/Theme.ts

```diff
@@ -380,12 +398,31 @@ export default class Theme {
     };
     public static serveTheme = async options => {
         try {
+            const currentContext = getActiveContext();
             const isSSR =
                 typeof options['ssr'] === 'boolean'
                     ? options['ssr']
                     : options['ssr'] == 'true'
                     ? true
                     : false;
+            if(!isSSR){
+                const questions = [
+                    {
+                        type: 'text',
+                        name: 'themeServe',
+                        message: `Do you wish to serve theme on ${chalk.blue(currentContext.domain)} of ${chalk.blue(currentContext.env)} env it will be affected with the new changes type YES to move forward?`,
+                    },
+                ]; {
+                    await inquirer.prompt(questions).then(async answers => {
+                        if (answers.themeServe === 'YES') {
+                            Logger.success('theme serving started');
```

Comment:
```suggestion
                            Logger.success('[Not recommended] SSR is disabled.');
```

Also remove log from line number 440. [Here](https://github.com/gofynd/fdk-cli/blob/e9415174ec7f7f3d1250d71549138c6b40c250a6/src/lib/Theme.ts#L440)